apis/v1alpha2/policy_types.go

```diff
@@ -40,6 +40,31 @@ type PolicyTargetReference struct {
 	Namespace *Namespace `json:"namespace,omitempty"`
 }
 
+// DirectPolicyTargetReference identifies an API object to apply direct policy to. This
+// should be used as part of Policy resources that can target Gateway API
+// resources. For more information on how this policy attachment model works,
```

Comment:
Small nit, since it's not only Gateway API resources you can target.
```suggestion
// should be used as part of Policy resources that can target single
// resources. For more information on how this policy attachment model works,
```